react/country/CHL.js

```diff
@@ -506,6 +525,25 @@ export default {
       valueIn: 'long_name',
       types: ['postal_code'],
       required: false,
+      handler: address => {
+        if (
+          !address.state?.value ||
+          !address.neighborhood?.value
+        ) {
+          return address
+        }
+
+        if (
+          countryData[address.state.value] &&
+          countryData[address.state.value][address.neighborhood.value]
+        ) {
```

Comment:
Well, if you can use optional chaining (like lines [530](https://github.com/vtex/address-form/pull/340/files#diff-a5e922924a10360dcff919c05d716b405072ba16eae7a1b18983803416e7732bR530) and [531](https://github.com/vtex/address-form/pull/340/files#diff-a5e922924a10360dcff919c05d716b405072ba16eae7a1b18983803416e7732bR531) you could do something like:

```suggestion
        if (countryData[address.state.value]?.[address.neighborhood.value]) {
```

but def not blocking.